### DISCUSSION TITLE
Is there a ceiling option? + 3D popup Apple

### DISCUSSION THREAD

**Discussion by divclass1** [google/model-viewer#4839] [Q&A]
Hi,

We have floor and wall placement, which is awesome!! But do we also have a ceiling placement? If not, where can I request a feature idea? I think this will be very cool and helpful.

By the way, on the latest iOS this annoying popup shows when using AR. This is not because of Modelviewer, but just an iOS update. I think this makes the AR experience a bit worse, it's an extra step users should take. Are there workarounds to avoid this? If not, maybe we can report a feature request on Apple.com to fix this? Apple always said that they believe in AR, so a good UX is part of that 😄 

![IMG_1833](https://github.com/user-attachments/assets/689ffecb-2290-40d5-8bd8-1418cfe615a9)

**Comment by elalish** [comment]
We don't support ceiling placement yet, but we could. PRs welcome! As for Apple's bug, FYI @AdaRoseCannon, though I expect the answer will be to submit it via their Feedback Assistant.

**Reply by divclass1** [reply]
Thanks!! I appreciate what you're doing by the way, and how fast you always reply🙌
